store_data.js

```diff
@@ -0,0 +1,23 @@
+const form2 = document.querySelector('#submit_form');
+const x = document.querySelector('#name');
+const email = document.querySelector('#email');
+const messg = document.querySelector('#msg');
+
+form2.addEventListener('submit', () => {
+  const obj = {
+    fullname: document.querySelector('#name').value,
+    EmailAdress: document.querySelector('#email').value,
+    message: document.querySelector('#msg').value,
+  };
+
+  localStorage.setItem('DATA', JSON.stringify(obj));
+});
+
+const getData = localStorage.getItem('DATA');
+const getDataValue = JSON.parse(getData);
+
+window.addEventListener('load', () => {
+  x.value = getDataValue.fullname;
```

Comment:
- [ ] Now JS is crushing, because you have an event listener `window.addEventListener('load',..` but there is nothing in the local storage when user open the page first time, its empty, therefore
`x.value, email.value, messg.value` are undefined and JS just does not know what you are asking it to do. To fix this, please add a validation to check if the local storage has any data

If you want to simulate an empty local storage and test your app, in console write `localStorage.clear()` this will clear all the data from local storage 😉 

![image](https://user-images.githubusercontent.com/77038610/146588002-3c37944d-5d5c-4396-ab2e-796190bfd7d5.png)